cunumeric/fft/fft.py

```diff
@@ -51,6 +51,8 @@ def _operate_by_axes(a, axes):
 @add_boilerplate("a")
 def fft(a, n=None, axis=-1, norm=None):
     """
+    fft(a, n=None, axis=1, norm=None)
```

Comment:
-1 ? I'm also not sure this stub is necessary if we aren't customizing anything,